drivers/wifi/nrf700x/Kconfig.nrf700x

```diff
@@ -17,7 +17,7 @@ config WIFI_NRF700X
 
 config NRF_WIFI_LOW_POWER
 	bool "Enable low power mode in nRF Wi-Fi chipsets"
-	default y
+	default n
```

Comment:
As `default n` is implicit choice, we typically just skip the `default` entry in such case.